validator/client/beacon-api/rest_handler_client_test.go

```diff
@@ -36,7 +36,7 @@ func TestGet(t *testing.T) {
 	mux.HandleFunc(endpoint, func(w http.ResponseWriter, r *http.Request) {
 		marshalledJson, err := json.Marshal(genesisJson)
 		require.NoError(t, err)
-
+		assert.Equal(t, "PrysmVC/Unknown/Local build", r.Header.Get("User-Agent"))
```

Comment:
the tests you added will fail if we build with the flag `--config=release`, which will set the git tag and commit to correct values. 
you should check the value from the `version.BuildValidatorData()` itself:
```suggestion
		assert.Equal(t, version.BuildValidatorData(), r.Header.Get("User-Agent"))
```